fuel/src/main/kotlin/com/github/kittinunf/fuel/core/Request.kt

```diff
@@ -131,7 +131,7 @@ class Request : Fuel.RequestConvertible {
         return this
     }
 
-    fun source(source: (Request, URL) -> File): Request {
+    fun sources(source: (Request, URL) -> Array<File>): Request {
```

Comment:
What do you think about return this as `Iterable<File>`? This might make our API more flexible.